src/Sql/dbo/Stored Procedures/OrganizationUser_DeleteByIds.sql

```diff
@@ -25,22 +25,74 @@ BEGIN
         EXEC [dbo].[SsoUser_DeleteMany] @UserAndOrganizationIds
     END
 
-    DELETE CU
-    FROM
-        [dbo].[CollectionUser] CU
+    
+    DECLARE @BatchSize INT = 100
+
+-- Delete SSO Users
+WHILE @BatchSize > 0
+    BEGIN
+    BEGIN TRANSACTION SsoUser_DeleteMany_SsoUsers
+
+    DELETE TOP(@BatchSize) SU
+        FROM
+        [dbo].[SsoUser] SU
     INNER JOIN
-        @Ids I ON I.Id = CU.OrganizationUserId
+        #SSOIds ON #SSOIds.Id = SU.Id
```

Comment:
I may have missed it, but I don't see where this temp table is created in the scope of this stored procedure or larger context.

Reply:
I see it now that I get further in the review, so, while `[dbo].[SsoUser_DeleteMany]` does create the temp table, `#SSOIds`, it does so within the scope of a committed transaction, before you open this transaction... not sure how that would interplay between the transaction scope across 2 different contexts (2 sprocs) and transactions (2 transactions), MSSQL can be funny that way.

Are you sure this is working as expected? I would expect, if I'm reading things correctly, this table at this point, `#SSOIds` is always going to be empty because it is out of context, but I'm not entirely certain, it's just one of those funny feelings (I could be wrong since there's the larger originating context of the parent stored procedure that started the call chain).

Reply:
oop that's embarrassing. This is some remnant of copying code for a template to batch my deletes below. No SSO deletion should be happening in this sproc!

Thank you for catching this!